### PR TITLE
201x/tests/objectConfiguration

### PR DESCRIPTION
**Pull request by AMR-**
Suggestions for a new test: Replicating an object configuration

In this task, Pepper is allowed to view a configuration of objects. A human in a different room (or separated by a screen) has access to all of these object arranged arbitrarily. First, Pepper must describe the scene in front of it and instruct the human how to arrange their objects in an identical manner. Second, Pepper moves to the human’s area and views their setup. If the setup is wrong in
some fashion, Pepper will instruct the human to make the appropriate changes.

Details:
When the task begins, Pepper can view objects in front of it. Objects can include furniture such as tables, chairs, and stools, as well as smaller objects such as food containers and blocks. Objects could differ in size, color, and shape. Pepper will describe what is sees, using relational language. For example, “Place the green can on top of the yellow plate that is in the center of the table.” This first part of the task is limited to 5 minutes. Afterwards, a tournament official will go to the human’s configuration and introduce at least one additional error that Pepper will have to correct. At this time, Pepper is led to the human’s area for the second part of the task. After viewing the scene, Pepper should assess the errors present in the configuration and give additional instructions to fix them, such as “There should not be a coke bottle in this scene” or “The water bottle should be on the red plate, not the yellow plate.” Successive rounds might have increasing number of objects.

Scoring
Pepper will have 5 minutes for the first phase (description) and 5 minutes for the second phase (critiquing the human’s results). Teams will be scored according to the number of objects correctly placed in the first phase, both in terms of absolute position and in relationship to each other, and the number of successful corrections in the second phase

### CODE REVIEW

File: tests/ObjectConfiguration.tex [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+\begin{itemize}
+	\item \textbf{Robot Starting Room}: any room.
+	\item \textbf{Human Starting Room}: a room adjacent to Robot Starting Room.
+	\item \textbf{Human Collaborator}: The human collaborator is in the Human Starting Room.
+	\item \textbf{Robot Objects} 10 objects are arranged in a space in front of the robot, potentially on top of a table or other furniture.
+	\item \textbf{Human Objects:} In this room there are the identical objects as in the other room, but arranged differently.
+\end{itemize}
+
+\subsection{Task Part 1}
+
```
Comment (kyordhel) [inline tests/ObjectConfiguration.tex]: Task requires complete redesign. It is too fine-grained and the goal of each of the steps is underspecified. It is unclear where the collaboration is leading to.

File: scoresheets/ObjectConfiguration.tex [hunk 2]
```diff
@@ -0,0 +1,27 @@
+The maximum time for this test is 6 minutes.
+
+\begin{scorelist}
+	\scoreheading{Describing the objects} % 100
+	%\scoreitem[n]{50 \/ n}{Description touches upon each of the n objects}
+	\scoreitem[10]{5}{Description touches upon each of the n objects}
```
[the remark this answers is not in the document]
Comment (kyordhel) [reply scoresheets/ObjectConfiguration.tex]: I don't think is a good idea to have $n$ in any scoresheet. All teams should be able to score the same.

[the remark this answers is not in the document]
Comment (AMR-) [reply scoresheets/ObjectConfiguration.tex]: the original idea was to have n objects, with each being worth (50/n) points, so that the "Describing the objects" in total is worth 50 points always, and n constant for the competition.

But perhaps having a constant value is better, so I changed n to 10.   Feel free to let me know which you like better

File: Rulebook.tex [hunk 3]
```diff
@@ -133,7 +133,7 @@ \chapter{Tests in Stage I}
 Meanwhile, the integration test has been designed to evaluate how this abilities work together while solving a common task.
 
 The total score for ability and integration tests is the average of the best two performances out of preferably three performances (given the time constraints of a competition).
-The point of this is the both elimate good and bad luck for the robots/teams and to get a more objective view of the performance,
```
[the remark this answers is not in the document]
Comment (AMR-) [reply Rulebook.tex]: i'll make a different PR for the typo correction

File: tests/ObjectConfiguration.tex (hunk 1 above)
Comment (AMR-) [reply tests/ObjectConfiguration.tex]: By "unclear where the collaboration is leading to" do you mean you would like there to be more of a story justification?

File: Rulebook.tex [hunk 4]
```diff
@@ -133,7 +133,7 @@ \chapter{Tests in Stage I}
 Meanwhile, the integration test has been designed to evaluate how this abilities work together while solving a common task.
 
 The total score for ability and integration tests is the average of the best two performances out of preferably three performances (given the time constraints of a competition).
-The point of this is the both elimate good and bad luck for the robots/teams and to get a more objective view of the performance, 
+The point of this is the both eliminate good and bad luck for the robots/teams and to get a more objective view of the performance,
```
[the remark this answers is not in the document]
Comment (AMR-) [reply Rulebook.tex]: will make new PR

File: scoresheets/ObjectConfiguration.tex [hunk 5]
```diff
@@ -0,0 +1,27 @@
+The maximum time for this test is 6 minutes.
```
[the remark this answers is not in the document]
Comment (AMR-) [reply scoresheets/ObjectConfiguration.tex]: done

File: scoresheets/ObjectConfiguration.tex [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+	\scoreitem[10]{5}{Description touches upon each of the n objects}
+	\scoreitem[10]{5}{Description of each object is accurate}
+
+	\scoreheading{Move to Part 2} % 30
+	\scoreitem{30}{Follow official to the human room}
+
+	\scoreheading{Correct Human Error} % 130
+	\scoreitem{10}{Focus on the correct space}
+	\scoreitem[4]{13}{Identifies each of the p errors}
+	\scoreitem[4]{12}{Has an accurate correction for each of the p errors}
```
[the remark this answers is not in the document]
Comment (AMR-) [reply scoresheets/ObjectConfiguration.tex]: added in commit 3 (dde38a4)

File: tests/ObjectConfiguration.tex [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+\begin{enumerate}
+
+	\item \textbf{Analyzing:} Robot views the object configuration in front of it.
+
+	\item \textbf{Description:} Robot describes the configuration to the human collaborator.  The robot can use relational languae or any other means of description.
```
[the remark this answers is not in the document]
Comment (AMR-) [reply tests/ObjectConfiguration.tex]: fixed in commit 3 (dde38a4)

File: tests/ObjectConfiguration.tex (hunk 1 above)
Comment (kyordhel) [reply tests/ObjectConfiguration.tex]: The test is extremely hard to follow because there is no intuitive goal. *Serve Breakfast*, *Cocktail Party*, *Follow Me*, *Fetch and Carry*, etc., give you a clue about the task and goal from the very title. Any person can easily grasp the context and what's the ultimate goal. In this test I couldn't find a direct and intuitive application for the task, something that people in the audience can easily relate to something they do in daily life.

The closest thing that came to my mind was when I move all objects from one surface to another while cleaning and then I can't remember what was the original configuration. Here It would be useful if the robot reminds me whether my ancient *Ming pot* was left or right of the stand with the *Scroll of Truth*, stuff that is most valuable to me and the robot is not allowed to touch. However:
- I am in the same room
- I am interacting with exactly the same objects
- Either I ask the robot or the robot realizes the mistake.
Therefore, I see no reason why I should replicate a setup that is in another room, nor why should I receive instructions from anyone (human or robot, that's extremely annoying).

However, I am not sure if what I mentioned above actually has something to do with the proposed test based only in the content of the tex file.

Designing a test is a very difficult task for it needs to be cross-culture understandable, so the first advice is give your test *as is* to read to a 3rd person (better if outside the work group) and ask them to explain back to you. Don't explain, just take notes and ask why that person makes that interpretation.

File: score_sheets.tex [hunk 8]
```diff
@@ -161,6 +161,12 @@
 \input{scoresheets/EEGPSR.tex}
 \end{scoresheet}
 
+% Object Configuration
```
Comment (kyordhel) [inline score_sheets.tex]: Wrong place. The test is a Stage I test and is located between EEGPSR and Open Challenge which are Stage II

File: scoresheets/ObjectConfiguration.tex [hunk 9]
```diff
@@ -2,23 +2,23 @@
 
 \begin{scorelist}
 	\scoreheading{Describing the objects} % 100
-	%\scoreitem[n]{50 \/ n}{Description touches upon each of the n objects}
-	\scoreitem[10]{5}{Description touches upon each of the n objects}
+	\scoreitem[10]{5}{Description touches upon each of the 10 objects}
```
Comment (kyordhel) [inline scoresheets/ObjectConfiguration.tex]: I find "Description touches upon" an unusual construct and its sense is quite ambiguous. Scoresheets must be straightforward. Of course if task changes, this needs also to change.

File: scoresheets/ObjectConfiguration.tex [hunk 10]
```diff
@@ -2,23 +2,23 @@
...
 	\scoreitem[10]{5}{Description of each object is accurate}
 
 	\scoreheading{Move to Part 2} % 30
 	\scoreitem{30}{Follow official to the human room}
 
 	\scoreheading{Correct Human Error} % 130
 	\scoreitem{10}{Focus on the correct space}
-	\scoreitem[4]{13}{Identifies each of the p errors}
+	\scoreitem[4]{13}{Identifies each of the errors}
```
Comment (kyordhel) [inline scoresheets/ObjectConfiguration.tex]: Remove each, you are scoring once per error, so it should be something like _Clearly state the nature of an error_. Plus, I feel 4x13 leading to a weird number.

File: tests/ObjectConfiguration.tex [hunk 11]
```diff
@@ -1,6 +1,6 @@
 \section{Replicating a Configuration of Objects}
 
-The robot has to understand relationships between objects, instruct a human to replicate that configuration of objects, and fix errors that the human makes.
+The robot has to understand relationships between objects, instruct a human operator to replicate that configuration of objects, and fix all errors made by the operator.
```
Comment (kyordhel) [inline tests/ObjectConfiguration.tex]: Understand is a strong word here. I don't understand why the pots in my mother balcony are in certain order, but I am able to remember and replicate its configuration. However, I do understand why fork and knife are located the way they are in a table setup so I can explain it.

File: tests/ObjectConfiguration.tex [hunk 12]
```diff
@@ -21,7 +21,7 @@ \subsection{Task Part 1}
 
 	\item \textbf{Analyzing:} Robot views the object configuration in front of it.
 
-	\item \textbf{Description:} Robot describes the configuration to the human collaborator.  The robot can use relational languae or any other means of description.
+	\item \textbf{Description:} Robot describes the configuration to the human collaborator.  The robot can use relational language or any other means of description.
```
Comment (kyordhel) [inline tests/ObjectConfiguration.tex]: Double space after the dot.
I haven't clear what is "relational language", meaning that I, as a human, couldn't execute the task without further explanation.

**Comment by kyordhel** [comment]
This proposal would need to be reworked to match 2019+ schema
